src/learning_service/requirements.txt

```diff
@@ -81,15 +80,12 @@ mypy==0.931
 mypy-extensions==0.4.3
 nanotime==0.5.2
 networkx==2.6.3
-nltk==3.7
```

Comment:
NLTK is used in the learning service, why is it removed from the requirements.txt? Is it transitive dependency of some other library?

Reply:
It's defined earlier in the same file.